### PR TITLE
The app was crashing after I solved the merge conflicts. Fixed it. Also cleaned up the navigation drawer code a bit.

### PR DESCRIPTION
**Pull request by Adithya2907**
### Description
Fixed crashing of app due to merging to two PRs.

Please ensure the following and put a check accordingly

- [ ] Ran checks with `./gradlew ktlintCheck` command

- [ ] Squashed multiple commits (_if irrelevant_) into one commit

### CODE REVIEW

File: app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt [hunk 1]
```diff
@@ -12,40 +12,36 @@ import club.devsoc.scanf.databinding.ActivityMainBinding
 import com.google.android.material.bottomsheet.BottomSheetBehavior
 import android.view.MenuItem
 import androidx.appcompat.app.ActionBarDrawerToggle
-import androidx.drawerlayout.widget.DrawerLayout
-import club.devsoc.scanf.R
 import com.google.android.material.navigation.NavigationView
 
-class MainActivity : AppCompatActivity(), View.OnClickListener {
+class MainActivity : AppCompatActivity(), View.OnClickListener, NavigationView.OnNavigationItemSelectedListener {
```
Comment (devansh-299) [inline app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt]: @Adithya2907 currently it's not enforced in the build checks to limit max characters to 100 but it is advised not to go beyond this. Please update the PR accordingly and resolve the merge conflicts

**Review by devansh-299 — CHANGES_REQUESTED** [review]

File: app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt (hunk 1 above)
Comment (Adithya2907) [reply app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt]: @devansh-299 I've made the change, and also resolved the merge conflicts. Is there any specific reason as to why code must not exceed 100 characters in a line?

Comment (devansh-299) [reply app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt]: @Adithya2907 There's no specific reason other than it makes code more readable. Different limits are used in different organizations like many use a limit of 80. I have seen 100 being adopted by many other organizations that's why I wanted to enforce this limit.
Also please squash your commits into one commit.

Comment (Adithya2907) [reply app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt]: Oh alright. And to squash commits into one, do I need to send a new PR?

Comment (devansh-299) [reply app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt]: > Oh alright. And to squash commits into one, do I need to send a new PR?

Nope, try to do this within this PR only

Comment (Adithya2907) [reply app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt]: @devansh-299 I'm not sure how I should squash commits into one. Do you have any resources which I can refer to?

**Pull request closed without merging**